example/sl-prediction/normalize_embedding.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+# Load required modules
+import sys, os, logging, numpy as np
+this_dir = os.path.dirname(__file__)
+sys.path.append(os.path.join(this_dir, '../../src'))
+
+# CORAL algorithm (Sun, Feng, & Saenko, arXiv:1612.01939v1)
+def coral(D_S, D_T, source_to_target=True, lamb=0.00001):
+    """
+    Arguments:
+    - D_S: Source data (numpy.array)
+    - D_T: Target data (numpy.array)
+    - source_to_target: Transform source into target space (bool)
+    Returns:
+    - D_S^*: Adjusted source data
+    """
+    # Set up
+    from sklearn.preprocessing import scale
+    from scipy.linalg import sqrtm
+    from numpy.linalg import pinv, inv
+    C_S = np.cov(D_S.T) + lamb*np.eye(D_S.shape[1])
+    C_T = np.cov(D_T.T) + lamb*np.eye(D_T.shape[1])
+    C_S_centered = scale(D_S, with_std = False)
+    C_T_centered = scale(D_T, with_std = False)
```

Comment:
Doesn't coral require both matrices to be of unit variance? If we want to not change the variance of the matrix being projected on to, we might have to think more carefully about how to scale the variances.